hwt905_ros_driver:
  ros__parameters:
    serial_port: "/dev/ttyUSB0"
    baud_rate: 115200
    frame_id: "imu_link"
    imu_topic: "imu"
